docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Troubleshooting kubeadm
+---
+
+{% capture overview %}
+
+Sometimes you will run into errors installing or operating kubeadm. Below we have listed 
+common failure scenarios and have provided steps that will help you to understand and hopefully
+fix the problem.
+
+If your problem is not listed below, please follow the following steps:
+
+- If you think your problem is a bug with kubeadm:
+  - Go to github.com/kubernetes/kubeadm and search for existing issues.
+  - If no issue exists, please [open one](https://github.com/kubernetes/kubeadm/issues/new) explaining
+    what the issue is, and what you expect kubeadm to do.
+
+- If you are unsure about how kubeadm or kubernetes works, and would like to receive 
+  support about your question, please open a question on StackOverflow. Please include 
+  relevant tags like `#kubernetes` and `#kubeadm` so folks can help you.
+
+If your cluster is in an error state, you may have trouble in the configuration if you see Pod statuses like `RunContainerError`,
+`CrashLoopBackOff` or `Error`. If this is the case, please read below.
+
+{% endcapture %}
+
+#### `ebtables` or `ethtool` not found during installation
```

Comment:
Can you make this generic, but illustrate the problem with ebtables and ethtool please?